include/gtfs/csv_parser.h

```diff
@@ -106,8 +107,11 @@ namespace gtfs {
 
         std::istringstream header_stream(header_line);
         std::string column;
-        while(std::getline(header_stream, column, ','))
+        while(std::getline(header_stream, column, ',')) {
```

Comment:
Yeah, good catch! Added support for `_getline` to take a custom delimeter, and moved over a few usages in `csv_parser.h`